optimade/server/mappers/entries.py

```diff
@@ -34,10 +36,24 @@ def all_aliases(cls) -> Tuple[Tuple[str, str]]:
                 (f"_{CONFIG.provider.prefix}_{field}", field)
                 for field in CONFIG.provider_fields.get(cls.ENDPOINT, [])
             )
+            + tuple(
+                (f"_{CONFIG.provider.prefix}_{field}", field)
+                for field in cls.PROVIDER_FIELDS
+            )
             + tuple(CONFIG.aliases.get(cls.ENDPOINT, {}).items())
             + cls.ALIASES
         )
 
+    @classmethod
+    def all_length_aliases(cls) -> Tuple[Tuple[str, str]]:
```

Comment:
Please add some doc strings - could just be copies or references to the similar methods.